tests/unit/test_init_plugin_no_extras.py

```diff
@@ -0,0 +1,45 @@
+"""Tests for init_plugin.py when no extra dependencies are installed."""
+
+import pytest
+from starlite import Starlite
+
+from starlite_saqlalchemy import init_plugin
+from starlite_saqlalchemy.exceptions import MissingDependencyError
+
+
+@pytest.mark.parametrize(
+    ("enabled_config", "error"),
+    [
+        ("do_cache", r"^.*\"redis\" is not installed.*$"),
+        ("do_sentry", r"^.*\"sentry_sdk\" is not installed.*$"),
+        ("do_worker", r"^.*\"saq\" is not installed.*$"),
+    ],
+)
+def test_extra_dependencies_not_installed(enabled_config: str, error: str) -> None:
```

Comment:
Instead of relying on `--ignore tests/unit/test_init_plugin_no_extras.py` in tox config, I'd rather that we skip this test based on environment so that simply calling `pytest` at the command line would still work.
```suggestion
from starlite_saqlalchemy import settings

SKIP = any([settings.IS_SAQ_INSTALLED, settings.IS_SENTRY_SDK_INSTALLED, ...])

@pytest.mark.skipif(SKIP, reason="test will only run if no extras are installed")
@pytest.mark.parametrize(
    ("enabled_config", "error"),
    [
        ("do_cache", r"^.*\"redis\" is not installed.*$"),
        ("do_sentry", r"^.*\"sentry_sdk\" is not installed.*$"),
        ("do_worker", r"^.*\"saq\" is not installed.*$"),
    ],
)
def test_extra_dependencies_not_installed(enabled_config: str, error: str) -> None:
```

Reply:
Agreed!